api/pytest.ini

```diff
@@ -4,5 +4,12 @@ DJANGO_SETTINGS_MODULE = conf.settings
 pythonpath = .
 
 filterwarnings=
+# Ignore warnings related to unverified HTTPS requests.
```

Comment:
@dhruvkb , could you please take a look at this PR? Do we still need to suppress this warning?